metabase/metabase
# Add a tooltip for disabled menu items in collections

Epic https://github.com/metabase/metabase/issues/23222

How to test:
- Create a SQL question `SELECT {{text}}`
- Pin this question in a collection
- Open the menu for this question and hover over the disabled item
- Make sure there is a tooltip describing the issue

<img width="721" alt="Screenshot 2022-07-25 at 18 37 39" src="https://user-images.githubusercontent.com/8542534/180818818-a5a19f14-89fd-4139-ae6f-9df3e135d813.png">
